### PR TITLE
MEN-4256: Mender-convert no longer installs the client by default

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Review by lluiscampos — APPROVED** [review]

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Missing: to add the new variable in https://docs.mender.io/system-updates-debian-family/variables#mender_client_version

**Comment by oleorhagen** [comment]
> Missing: to add the new variable in https://docs.mender.io/system-updates-debian-family/variables#mender_client_version

Oh yes, good catch! I didn't know we documented these

File: 04.System-updates-Debian-family/99.Variables/docs.md [hunk 1]
```diff
@@ -109,6 +109,12 @@ Set the default index for the boot partition.
 The size of the boot partition.
 
 
+#### `MENDER_CLIENT_INSTALL`
+
+> Value: y (default)
```
Comment (lluiscampos) [inline 04.System-updates-Debian-family/99.Variables/docs.md:114]:
```suggestion
> Value: y(default)/n
```

See the syntax in, for example, `MENDER_ADDON_CONNECT_INSTALL`

**Review by lluiscampos — APPROVED** [review]
See comment below, though.

File: 04.System-updates-Debian-family/99.Variables/docs.md (hunk 1 above)
Comment (oleorhagen) [reply 04.System-updates-Debian-family/99.Variables/docs.md:114]: This is a bit all over the place though:

```
#### `MENDER_DATA_PART_GROWFS`

> Values: y(default)/n

Enable/Disable automatically growing the filesystem to fill the physical storage device.
```

```
#### `MENDER_ADDON_CONFIGURE_INSTALL`

> Value: y/n(default)

Install the Mender Configure add-on.

```
```
#### `MENDER_BOOT_PART_SIZE_MB`

> Value: 40 (default)

The size of the boot partition.

``` (with air, like most of them have)

To be honest, it's the `ADDON` values standing out as the outliers

Comment (lluiscampos) [reply 04.System-updates-Debian-family/99.Variables/docs.md:114]: They all look good to me.

Criteria seems to be: we use `/` to describe legal values with no air, adding `(default)` on the default one, and we have air on the "any-string" kind of config.

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**